website/docs/use_cases.md

```diff
@@ -6,12 +6,14 @@ toc: true
 last_modified_at: 2019-12-30T15:59:57-04:00
 ---
 
-## Near Real-Time Ingestion
+Apache Hudi provides the foundational features required to build a state-of-the-art Lakehouse. 
+The following are examples of use cases for why many choose to use Apache Hudi:
 
-Hudi offers some great benefits across ingestion of all kinds. Hudi helps __enforces a minimum file size on DFS__. This helps
-solve the ["small files problem"](https://blog.cloudera.com/blog/2009/02/the-small-files-problem/) for HDFS and Cloud Stores alike,
-significantly improving query performance. Hudi adds the much needed ability to atomically commit new data, shielding queries from
-ever seeing partial writes and helping ingestion recover gracefully from failures.
+## A Streaming Data Lake
+As outlined in depth in this blog post, https://hudi.apache.org/blog/2021/07/21/streaming-data-lake-platform, Apache Hudi 
```

Comment:
[Just a suggestion] Rephrase like this ?
`Apache Hudi is a Streaming Data Lake Platform that unlocks near real-time data ingestion and incremental processing pipelines with ease. This blog post outlines in depth on this use case - https://hudi.apache.org/blog/2021/07/21/streaming-data-lake-platform
`

Reply:
good call, i adjusted